core/src/main/java/ai/timefold/solver/core/impl/domain/common/accessor/ReflectionMethodMemberAccessor.java

```diff
@@ -19,6 +19,10 @@ public final class ReflectionMethodMemberAccessor extends AbstractMemberAccessor
     private final MethodHandle methodHandle;
 
     public ReflectionMethodMemberAccessor(Method readMethod) {
+        this(readMethod, true);
+    }
+
+    public ReflectionMethodMemberAccessor(Method readMethod, boolean requiredReturnType) {
```

Comment:
I think `returnTypeRequired` makes it more obvious that it's a `boolean`.
Please apply consistently throughout the codebase.